### ISSUE TITLE
psd not openning

### ISSUE THREAD

**Issue by dmit8815** [avocode/avocode#382]
IMPORTANT: Please try to describe how did it happen.

psd uploaded, but i can't open it after that. I just see "generating image" notice
-----
App v.2.4.1-9839e14
Source UID: 98055
```
Error: Invariant Violation: Dispatch.dispatch(...): Cannot dispatch in the middle of a dispatch.
    at invariant (/home/watson/Avocode/resources/app.asar/node_modules/flux/lib/invariant.js:42:15)
    at o.Dispatcher.dispatch (/home/watson/Avocode/resources/app.asar/node_modules/flux/lib/Dispatcher.js:185:5)
    at o.handleViewAction (/home/watson/Avocode/resources/app.asar/node_modules/@avocode/core/lib/dispatcher.js:1:707)
    at t.module.exports.t.setDocument (/home/watson/Avocode/resources/app.asar/node_modules/@avocode/core/lib/active-document.js:1:1668)
    at t._fireNoActiveViewEvents (/home/watson/Avocode/resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:24433)
    at t._fireNoActiveViewEvents (/home/watson/Avocode/resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:999)
    at atom-workspace.<anonymous> (/home/watson/Avocode/resources/app.asar/node_modules/@avocode/core/lib/avocode.js:1:8427)
    at atom-workspace.handler (/home/watson/Avocode/resources/app.asar/src/space-pen-extensions.js:110:34)
    at atom-workspace.jQuery.event.dispatch (/home/watson/Avocode/resources/app.asar/node_modules/space-pen/vendor/jquery.js:4681:9)
    at atom-workspace.elemData.handle (/home/watson/Avocode/resources/app.asar/node_modules/space-pen/vendor/jquery.js:4359:46)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/home/watson/Avocode/resources/app.asar/src/command-registry.js:251:29)
    at CommandRegistry.handleCommandEvent (/home/watson/Avocode/resources/app.asar/src/command-registry.js:3:61)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/home/watson/Avocode/resources/app.asar/src/command-registry.js:166:19)
    at PaneContainerView.jQuery.fn.trigger (/home/watson/Avocode/resources/app.asar/src/space-pen-
```

**Comment by rychlis** [comment]
Does the psd open after restarting Avocode?

**Comment by hog-seruj** [comment]
I have same issue, after last update. OS Ubuntu 14.01. Avocode 2.4.1

**Comment by rychlis** [comment]
@hog-seruj and does the psd open after resyncing or restarting Avocode or is the error persistent - happens every time you open the psd

**Comment by hog-seruj** [comment]
I can't load PSD

**Comment by hog-seruj** [comment]
Hello,

We noticed that our servers weren’t able to process your design ”01-HM-Objet-Connecte_page-edito_05.psd”. It’s probably because the design file is too large or too complex. Unfortunately, that’s an issue we’re currently working on.

In the meantime, you can use the Photoshop Plugin to sync this design. Get the plugin here: https://manager.avocode.com/downloads/

If you don't have Photoshop, you can ask someone in your team or we might be able to upload that PSD for you manually. Let us know if you need that.

Thank you,
– The Avocode Team

**Comment by hog-seruj** [comment]
File 137mb

**Comment by rychlis** [comment]
@hog-seruj did you get the same error as dmit8815 `Error: Invariant Violation: Dispatch.dispatch(...): Cannot dispatch in the middle of a dispatch.` ? Otherwise this problem is probably not really related.

**Comment by hog-seruj** [comment]
YEs. sorry my bad. Need to create new issue?

**Comment by rychlis** [comment]
@hog-seruj the reason your PSD failed is probably issue #152. It would be better to give us an email at team@avocode.com so we can solve this together individually.

**Comment by hog-seruj** [comment]
@rychlis, send mail. Pls check PSD.

**Comment by dmit8815** [comment]
@rychlis restarting Avocode not worked for me :(

Notice: PSD opens fine in Adobe Creative Cloud, but i don't want to use their instrument.. About big layers - i don't know, maybe i can send this PSD to you  so you can analyze it..

https://mrkr.io/XpJ0AOBFPz

**Comment by dmit8815** [comment]
I'm still experiencing problems uploading PSD. Is there any progress on this problem?

**Comment by rychlis** [comment]
@dmit8815 I am not sure about what kind of problem are you talking but the initial error you reported ("Invariant Violation") is fixed in Avocode 2.4.3. It seems that you don't have the latest version at the moment.

**Comment by rychlis** [comment]
I am closing the issue, will re-open if you still experience this in latest Avocode 2.5.2
